### DISCUSSION TITLE
Npm build issues

### DISCUSSION THREAD

**Discussion by litelime** [whyboris/Video-Hub-App#676] [Q&A] [answered]
Hello,
I am getting this issue with the current version of "main"
```
glob-parent  <5.1.2
Severity: moderate
Regular expression denial of service - https://npmjs.com/advisories/1751
No fix available
node_modules/webpack-dev-server/node_modules/glob-parent
  chokidar  1.0.0-rc1 - 2.1.8
  Depends on vulnerable versions of glob-parent
  node_modules/webpack-dev-server/node_modules/chokidar
    webpack-dev-server  2.0.0-beta - 3.11.2
    Depends on vulnerable versions of chokidar
    node_modules/webpack-dev-server
      @angular-devkit/build-angular  *
      Depends on vulnerable versions of webpack-dev-server
      node_modules/@angular-devkit/build-angular
```

Looks like glob-parent is giving me the issue and audit/audit --force are not able to fix it. 
Unfortunately I can't build with this error, any ideas on how to solve this? 
I would assume I need to change the version but I see it show up multiple times in the package file.

**Comment by litelime** [comment, marked as answer]
I completely deleted the repo and re-cloned, then did the npm install again. Still got the same audit warning but was able to successfully build now, so probably some other issue was causing the build failure before.

**Reply by whyboris** [reply]
Sorry I have to start checking the `Discussions` section more often 😓 I didn't see this thread until now (few months late).

I think with errors like this it sometimes helps to remove `node_modules` completely and sometimes it helps to also delete the `package-lock.json` file before `npm install`

I'm glad you found a work-around 😅

**Reply by whyboris** [reply]
ps - now because of some dependencies getting old, `npm install` errors out; but the error is easy to overcome with:

```sh
npm install --legacy-peer-deps
```
